envs/example/ci-full-rhel/group_vars/ceph_monitors.yml

```diff
@@ -1,12 +0,0 @@
----
```

Comment:
why is this file being removed?  It was added for reference even thou ci-full-rhel does not enable ceph.